src/client/index.tsx

```diff
@@ -1,18 +1,29 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
+interface ConsultantParameter {
+    label: string;
+    value: string;
+}
 
-const name = "John Doe";
+const consultantLines: ConsultantParameter[] = [{
+    label: "Name",
+    value: "John Doe",
+}]
+
+const consultantRow: JSX.Element = (
+    <div className={"tableRow"}>
+        <div className={"tableCell"}>{consultantLines[0].label}</div>
+        <div className={"tableCell"}>{consultantLines[0].value}</div>
+    </div>
+)
```

Comment:
Don't forget about `;` here and on `line 12`. If you use `=`  to declare something it should have semicolon in the end. That's why `line 7` doesn't need it.